collectors/channels_collector.go

```diff
@@ -174,10 +186,43 @@ func NewChannelsCollector(lnd lndclient.LightningClient, errChan chan<- error,
 			[]string{"amount"}, nil,
 		),
 
-		lnd:         lnd,
-		primaryNode: cfg.PrimaryNode,
-		errChan:     errChan,
+		lnd:                 lnd,
+		primaryNode:         cfg.PrimaryNode,
+		closedChannelsCache: []lndclient.ClosedChannel{},
```

Comment:
nit: no need to initialize as `nil` will cast to an empty slice.

Reply:
OK, fixing this up.